app/src/app/api/v0/inventory/[inventory]/progress/route.ts

```diff
@@ -0,0 +1,172 @@
+import UserService from "@/backend/UserService";
+import { db } from "@/models";
+import { logger } from "@/services/logger";
+import { apiHandler } from "@/util/api";
+import { NextResponse } from "next/server";
+
+import type { Sector } from "@/models/Sector";
+
+// sort whole inventory by GPC reference number
+function romanNumeralComparison(sectorA: Sector, sectorB: Sector) {
+  const a = sectorA.referenceNumber || "";
+  const b = sectorB.referenceNumber || "";
+
+  const romanTable: Record<string, number> = {
+    I: 1,
+    II: 2,
+    III: 3,
+    IV: 4,
+    V: 5,
+    VI: 6,
+    VII: 7,
+    "": 1337,
```

Comment:
will this be applicable?
MCCCXXXVII: 1337

Reply:
This is just a default case to prevent errors for empty strings (that can occur if there is no reference number for some reason). So I just have this case for an empty string so there is still a number and they can be sorted.